src/python/pants/backend/python/goals/run_python_source_integration_test.py

```diff
@@ -74,7 +138,7 @@ def my_file():
             """
         ),
         "src_root2/utils/BUILD": "python_sources()",
-        "src_root2/codegen/hello.proto": 'syntax = "proto3";\nmessage Hi {{}}',
+        "src_root2/codegen/hello.proto": 'syntax = "proto3";\nmessage Hi {string name = 1;}',
```

Comment:
I dunno what the difference is, but this change was necessary for protobuf to successfully codegen.